components/generic/ContentCard.vue

```diff
@@ -1,8 +1,10 @@
 <template>
   <b-card
-    class="mb-4 text-left"
+    class="mb-4 text-left content-card"
     data-qa="content card"
     no-body
+    :class="isRelated && 'related-card'"
```

Comment:
Can you explain whats going on here? 'related-card' will only show if 'isRelated' equals true?

Reply:
Yes, indeed. See also https://michaelnthiessen.com/conditional-class-binding-vue#guard-expressions

Reply:
I wonder if it's better to take the standard approach which I think is easier to read

`:class="{ 'related-card' : isRelated }"`